tests/python_client/testcases/test_search.py

```diff
@@ -428,6 +429,31 @@ def test_search_param_invalid_expr_bool(self, get_invalid_expr_bool_value):
                             check_items={"err_code": 1,
                                          "err_msg": "failed to create query plan"})
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_search_with_expression_invalid_like(self):
+        """
+        target: test search int64 and float with like
+        method: test search int64 and float with like
+        expected: searched failed
+        """
+        nb = 1000
+        dim = 8
+        collection_w, _vectors, _, insert_ids = self.init_collection_general(prefix, True,
+                                                                             nb, dim=dim,
+                                                                             is_index=True)[0:4]
+        index_param = {"index_type": "IVF_FLAT", "metric_type": "L2", "params": {"nlist": 100}}
+        collection_w.create_index("float_vector", index_param)
+        collection_w.load()
+        expression = ["int64 like 33", "float LIKE 33"]
```

Comment:
expression only supports string type value, if you enter a list, this case will check this point rather than "like" or "LIKE", so need to modify here.